### DISCUSSION TITLE
How to map CSV/Parquet columns names to atoti.Table column name?

### DISCUSSION THREAD

**Discussion by alexis-bvq** [atoti/atoti#524] [Q&A] [answered]
Hello, as of now it is not possible to read a csv with header name not matching the fields name of the table.
See example below:
```python
import pandas as pd
import atoti as tt
session = tt.create_session()
pd.DataFrame({
    "AA": ["a", "b"],
    "BB": [1.0, 2.0]
}).to_csv("./test.csv", index=False)

table = session.create_table(types={"aa": tt.type.STRING, "bb": tt.type.DOUBLE}, name="test")
table.load_csv("./test.csv")
table.head()
# aa bb
# N/A 0.0
# N/A 0.0
```
It would be nice to have a mapping parameter within the `load_csv` function in order to map the mismatching names.
I have a client who had to go through pandas to rename columns first and then load data using `load_pandas` which we could avoid.

**Comment by bloussou** [comment, marked as answer]
Hello,
In the next version you will have the ability to do so with a `columns` parameter.
In your case :
```python
table.load_csv("./test.csv", columns={"AA": "aa", "BB": "bb"})
```
see : https://github.com/atoti/atoti/issues/462
